eos-phone-home

```diff
@@ -109,10 +111,10 @@ class PhoneHome(object):
                 dt_data = self._lookup_or_get_variable('dt_info')
                 dt_fields = dt_data.split(',')
                 vendor = dt_fields[0]
-        except Exception as e:
-            print("Unable to get vendor name!", e, file=sys.stderr)
+        except:
+            log.exception("Unable to get vendor name!")
```

Comment:
It's just a shorthand for `log.error(..., exc_info=True, ...)` but why not save a few keystrokes :)